script/Ciel/Day 14/14_01_CIEL14_1.txt

```diff
@@ -516,6 +521,7 @@ I can't take it anymore.
 [sha:79bfacb831ef6ecdd70843c787034c64ab3d3a53]{
 -- Page 28, Offset 37097.
 -- 気絶する事さえ、できない。
+// Improve contrast / clarity?
```

Comment:
Man this one kinda sucks
> I can't even escape by fainting. 
> Even the release of unconsciousness is denied to me.

Reply:
I like the latter, but "release of consciousness" is probably too long.
>But even blacking out is denied to me.